pkg/client/postrenderer.go

```diff
@@ -0,0 +1,132 @@
+package client
+
+import (
+	"bytes"
+
+	sdkhandler "github.com/operator-framework/operator-lib/handler"
+	"helm.sh/helm/v3/pkg/action"
+	"helm.sh/helm/v3/pkg/kube"
+	"helm.sh/helm/v3/pkg/postrender"
+	"k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/cli-runtime/pkg/resource"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/yaml"
+
+	"github.com/operator-framework/helm-operator-plugins/internal/sdk/controllerutil"
+	"github.com/operator-framework/helm-operator-plugins/pkg/manifestutil"
+)
+
+func WithInstallPostRenderer(pr postrender.PostRenderer) InstallOption {
+	return func(i *action.Install) error {
+		i.PostRenderer = pr
+		return nil
+	}
+}
+
+func WithExtraInstallPostRenderer(pr postrender.PostRenderer) InstallOption {
```

Comment:
I wonder how soon someone will want an option like this which prepends a postrenderer 😆

Reply:
That's a good point. I'll at least rename this to `Append` rather than `WithExtra`. That way if we ever need prepend behavior, it fits a lot easier.